Build.ps1

```diff
@@ -117,7 +122,7 @@ Task PublishImpl -requiredVariables PublishDir {
     }
 
     Write-Output "Publishing $ModuleName"
-    Write-Output "Version is $env:GitVersion_Version (prerelease: $env:Prerelease)"
+    Write-Output "Version is $env:GitVersion_Version $(prerelease: $env:Prerelease)"
```

Comment:
`$(prerelease:` ?

Maybe the old line is still valid?

Reply:
Yes, my fuck up.